### PR TITLE
Fixed issue #1607 updated reame removed typos

### PR DESCRIPTION
**Pull request by Ankush1oo8**
Fixed Unclosed parentheses in README #1607

### CODE REVIEW

**Review by Kuuuube — APPROVED** [review]

**Comment by Ankush1oo8** [comment]
What to do about checks

**Comment by Kuuuube** [comment]
> What to do about checks

It's fine, sometimes that one fails since it sends out web requests. Doesn't look like anything is actually wrong.

**Comment by Ankush1oo8** [comment]
> > What to do about checks
> 
> It's fine, sometimes that one fails since it sends out web requests. Doesn't look like anything is actually wrong.
Okey So now it will be merged right

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 # Visit [yomitan.wiki](https://yomitan.wiki) to learn more!
 
-:wave: **Yomitan is [the successor](https://foosoft.net/posts/passing-the-torch-to-yomitan/) to Yomichan** ([migration guide](https://yomitan.wiki/yomichan-migration/) which was [sunset](https://foosoft.net/posts/sunsetting-the-yomichan-project/) by its owner on Feb 26, 2023. We have made a number of foundational changes to ensure **the project stays alive, works on latest browser versions, and is easy to contribute to**.
+:wave: **Yomitan is [the successor](https://foosoft.net/posts/passing-the-torch-to-yomitan/) to Yomichan** [migration guide](https://yomitan.wiki/yomichan-migration/) which was [sunset](https://foosoft.net/posts/sunsetting-the-yomichan-project/) by its owner on Feb 26, 2023. We have made a number of foundational changes to ensure **the project stays alive, works on latest browser versions, and is easy to contribute to**.
```
Comment (jamesmaa) [inline README.md:12]: Hey actually the fix is to complete the parentheses after "migration guide" rather than remove the parentheses. Can you fix this?

**Review by jamesmaa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jamesmaa) [reply README.md:12]: The sentence reads wrong if "migration guide" is not in a parentheses

Comment (Ankush1oo8) [reply README.md:12]: So should I place the parenthesis after the migration guide and link

Comment (jamesmaa) [reply README.md:12]: Yes notice how the sentence doesn't read well if the parentheses is removed
image

It should be:

> Yomitan is the successor to Yomichan (migration guide) which was sunset by its owner on Feb 26, 2023. We have made a number of foundational changes to ensure the project stays alive, works on latest browser versions, and is easy to contribute to.

Comment (Ankush1oo8) [reply README.md:12]: Okey

**Review by jamesmaa — APPROVED** [review]
Thanks!

**Pull request merged**